### DISCUSSION TITLE
I was asking about the design file

### DISCUSSION THREAD

**Discussion by amosmachora** [reactdeveloperske/reactdevske-website#134] [Q&A] [answered]
I want to contribute to this project but couldn't find the design file. If somebody could help me with the file I would be really grateful.

**Comment by orama254** [comment, marked as answer]
Greetings @amosmachora 

Checkout the Design files [HERE](https://www.figma.com/file/TVwnaDhBGeVdnVKdf6H91C/React-developers-community-website?node-id=741%3A976). Let me know if there is anything else you want assistance with in regards to the project.
